### ISSUE TITLE
question about LPG layer

### ISSUE THREAD

**Issue by derekwong66** [cogaplex-bts/bts#54]
Hi @cogaplex-bts 

thanks for ur effort. I am reading ur tensorflow pipeline. I got one question about LPG. As ur paper mentioned, n4 will go through singmoid and n1, n2, n3 will pass into tanh then L2_nor. Then concat them together. But in bts class, I found that 
            plane_eq_8x8 = self.reduction_1x1(daspp_feat, num_filters / 2)
            plane_normal_8x8 = tf.nn.l2_normalize(plane_eq_8x8[:, :, :, 0:3], axis=3)
            plane_dist_8x8 = plane_eq_8x8[:, :, :, 3]
            plane_eq_8x8 = tf.concat([plane_normal_8x8, tf.expand_dims(plane_dist_8x8, 3)], 3)
            depth_8x8 = lpg.local_planar_guidance(plane_eq_8x8, upratio=8, focal=self.focal)
            depth_8x8_scaled = tf.expand_dims(depth_8x8, 3) / self.max_depth
            depth_8x8_scaled_ds = self.downsample_nn(depth_8x8_scaled, 4)
plane_normal_8x8 did not pass into sigmoid.
same as plane_eq_8x8
So May I ask why?

Thanks,
Derek
